docs/data/toolpad/core/components/account/AccountCustomLocaleText.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { AuthenticationContext, SessionContext } from '@toolpad/core/AppProvider';
+import { AppProvider } from '@toolpad/core/AppProvider';
```

Comment:
This is a clean up on the demo (since none of the demos on this page were using the `Account` component inside the `AppProvider` - added that to one example to demonstrate a local override of the `localization`)